workspace/packages.dhall

```diff
@@ -0,0 +1,32 @@
+let upstream =
+      https://github.com/purescript/package-sets/releases/download/psc-0.15.4-20220901/packages.dhall
+        sha256:f1531b29c21ac437ffe5666c1b6cc76f0a9c29d3c9d107ff047aa2567744994f
+
+let additions =
+      { spec-reporter-codewars =
+        { repo = "https://github.com/andys8/purescript-spec-reporter-codewars"
+        , version = "e40d7b623fd52b336fb3007b9fefa8644ed45a55"
```

Comment:
:information_source: Spec reporter installed as dependency. Needs to be switched to `codewars` repository, once https://github.com/codewars/purescript-spec-reporter-codewars/pull/1 is merged

Reply:
81aca8fcce256d23ae3efa5c82caca2f14b002c8